okteto-compose.yml

```diff
@@ -66,6 +66,18 @@ services:
         FORWARD_REVIEW_PORT: 27017
         REVIEW_USERNAME: '${REVIEW_USERNAME:-citizix}'
         REVIEW_PASSWORD: '${REVIEW_PASSWORD:-S3cret}'
+    api-edge:
+      depends_on:
+        - api-instrument
+        - api-piece
+        - api-review
+      image: johantm/instrument_edge_api:latest     #does not exist yet
+      ports:
+        - '8050:8050'
+      environment:
+        INSTRUMENT_SERVICE_BASEURL: instrument-service:8051
+        PIECE_SERVICE_BASEURL: piece-service:8052
+        REVIEW_SERVICE_BASEURL: review-service:8053
```

Comment:
Waarschijnlijk goed als we deze environment variabelen ook in de dockerfile zetten. De push naar docker en okteto gebeurd trouwens enkel op push naar main.

Reply:
Ik denk trouwens ook dat de ports hier wegmogen